README.md

```diff
@@ -18,8 +18,6 @@ needs/deficiencies).
       credentials, or disk space issues, or time of day, etc.);
 * Available
    1. Jobs don’t wait unreasonable amounts of time in order to begin executing;
-   2. Relevant monitoring/telemetry (backend) services run at different layers
```

Comment:
Yeah, it makes sense to move this to observability. I guess my idea was host and job success metrics can inform decisions in order to support available systems. I might just drop this if it's going to move to the observability section, since it's redundant there.